<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HeatMethodWidget</class>
 <widget class="QDockWidget" name="HeatMethodWidget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>708</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>365</width>
    <height>708</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Heat Method</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="3" column="0">
     <layout class="QGridLayout" name="coloringGroup" columnstretch="50,50">
      <item row="2" column="0">
       <spacer name="verticalSpacer_4">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1" rowspan="3">
       <widget class="QScrollArea" name="scrollArea">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>161</width>
           <height>371</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0" alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="legendLabel">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>RAMP DISPLAYING</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="coloringChoiceGroup">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="title">
         <string>Color Ramp</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2" columnstretch="50,50">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="2" column="0">
          <widget class="QLabel" name="minColorLabel">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Min color</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="minColorButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="maxColorLabel">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Max color</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="maxColorButton">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="verticalSpacer">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QGroupBox" name="methodGroup">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Method</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <widget class="QComboBox" name="methodBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QGroupBox" name="extremeValueGroup">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Extreme Value</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="2">
        <widget class="QPushButton" name="zoomToMaxButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Zoom to max value</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="extremeValueMaxLabel">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Max Value</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="DoubleEdit" name="maxBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="11" column="0" colspan="2">
     <widget class="QPushButton" name="estimateDistancesButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="font">
       <font>
        <family>Cantarell</family>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Estimate Geodesic Distances</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="sourcesGroup">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="title">
       <string>Source Vertices</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QPushButton" name="createSourceVerticesButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="toolTip">
          <string extracomment="Use 'Shift + Left Click'"/>
         </property>
         <property name="toolTipDuration">
          <number>1</number>
         </property>
         <property name="text">
          <string>Create Source Vertices Selection Item</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DoubleEdit</class>
   <extends>QLineEdit</extends>
   <header>CGAL_double_edit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
